### PR TITLE
feat: add resource limits and requests to ocean controller

### PR DESCRIPTION
**Pull request by arifalkner**

### CODE REVIEW

**Review by Iady-m — APPROVED** [review]

File: ocean.tf [hunk 1]
```diff
@@ -111,5 +111,7 @@ module "ocean-controller" {
   controller_image    = var.controller_image
   node_selector       = var.controller_node_selector
   image_pull_policy   = var.image_pull_policy
+  resource_limits     = var.controller_resource_limits
```
Comment (liranp) [inline ocean.tf]:
```suggestion
  resources_limits     = var.controller_resources_limits
```

Comment (liranp) [reply ocean.tf]: [See](https://github.com/spotinst/terraform-spotinst-ocean-controller/blob/v0.38.0/variables.tf#L70-L78).

File: ocean.tf [hunk 2]
```diff
@@ -111,5 +111,7 @@ module "ocean-controller" {
...
+  resource_requests   = var.controller_resource_requests
```
Comment (liranp) [inline ocean.tf]:
```suggestion
  resources_requests   = var.controller_resources_requests
```

Comment (liranp) [reply ocean.tf]: [See](https://github.com/spotinst/terraform-spotinst-ocean-controller/blob/v0.38.0/variables.tf#L80-L88).

**Review by liranp — CHANGES_REQUESTED** [review]

File: ocean.tf [hunk 3]
```diff
@@ -111,5 +111,7 @@ module "ocean-controller" {
   controller_image    = var.controller_image
   node_selector       = var.controller_node_selector
   image_pull_policy   = var.image_pull_policy
+  resources_limits    = var.controller_resource_limits
```
Comment (liranp) [inline ocean.tf]:
```suggestion
  resources_limits    = var.controller_resources_limits
```

File: ocean.tf [hunk 4]
```diff
@@ -111,5 +111,7 @@ module "ocean-controller" {
...
+  resources_requests  = var.controller_resource_requests
```
Comment (liranp) [inline ocean.tf]:
```suggestion
  resources_requests  = var.controller_resources_requests
```

**Review by liranp — CHANGES_REQUESTED** [review]

**Review by liranp — APPROVED** [review]

**Pull request merged**